### ISSUE TITLE
Wine Compatability

### ISSUE THREAD

**Issue by akxks** [MobiusDevelopment/Tera-Launcher#1]
Hi i was wondering why the launcher just closes when logging in (I am using this on crossover) trying to make TERA work under Wine compatibility layer. This is the only launcher that does not show error but just closes, tried Neolauncher but that just shows a HTTP error (as it is not implemented in Wine) the way it authenticates.

And I also tried to look at Ragezone posts, some guy in 2019 made a TERA-EME launcher for Wine, however the github page is 404 down

Just trying to play this game and its private server on Mac. The DB,Website and Java Server for 1732 work running on macOS. Just trying to make the client and it's launcher work under a Windows 7 Bottle w/ DirectX / DXVK and vcruntime on Crossover app

**Comment by MobiusDevelopment** [comment]
Basically what it does is this line.
Process.Start("tera.exe", "1 " + getMD5(txtPassword.Text) + " 0 1 " + txtAccount.Text + " en")

**Comment by akxks** [comment]
thanks, why all the numbers ? Also, are you able to make a debug.txt to see why it just closes when logging in? For some reason TERA.exe didn't start when using your launcher, after I wrote the correct login details..

I'm wondering what to do and if the launcher is still passing wine compatability onto the other exe 

Also what is txtAccount.txt and am I able to write this on my own in the CrossOver run as command ..."exe location" + password

**Comment by MobiusDevelopment** [comment]
It is made for https://github.com/MobiusDevelopment/TJServer client

**Comment by akxks** [comment]
Yeah I know that, but I wasn't asking about the server I was asking about the client. I already have the Java server running perfectly and the website db and auth.php.
